README.md

```diff
@@ -0,0 +1,36 @@
+## Running tests
+Comming soon
+
+## Examples
+
+## Beta Execution
+
+##### Beta branch on amd64 vm
+DEVICE_USER=sergio-j-cazzolato DEVICE_PORT=8022 BRANCH=2.27.2 ~/workspace/snappy-qa-jobs/scripts/run_external_device.sh dev_snapd_amd64
```

Comment:
AFAIK we should be using the `release` branches, in other words the `release/2.27` branch instead of the `2.27.2` tag